Chap_API_Storage.tex

```diff
@@ -59,70 +59,70 @@ \section{Storage support constants}
 
 \begin{constantdesc}
 %
-\declareconstitemProvisional{PMIX_STORAGE_ACCESSIBILITY_NODE}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_NODE}{0x0000000000000001}
 The storage system resources are accessible within the same node.
 %
-\declareconstitemProvisional{PMIX_STORAGE_ACCESSIBILITY_SESSION}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_SESSION}{0x0000000000000002}
 The storage system resources are accessible within the same session.
 %
-\declareconstitemProvisional{PMIX_STORAGE_ACCESSIBILITY_JOB}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_JOB}{0x0000000000000004}
 The storage system resources are accessible within the same job.
 %
-\declareconstitemProvisional{PMIX_STORAGE_ACCESSIBILITY_RACK}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_RACK}{0x0000000000000008}
 The storage system resources are accessible within the same rack.
 %
-\declareconstitemProvisional{PMIX_STORAGE_ACCESSIBILITY_CLUSTER}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_CLUSTER}{0x0000000000000010}
 The storage system resources are accessible within the same cluster.
 %
-\declareconstitemProvisional{PMIX_STORAGE_ACCESSIBILITY_REMOTE}
+\declareconstitemvalueProvisional{PMIX_STORAGE_ACCESSIBILITY_REMOTE}{0x0000000000000020}
 The storage system resources are remote.
 %
 \end{constantdesc}
 
 \declarestruct{pmix_storage_persistence_t}
 \provisionalMarker{}
 
-The \refstruct{pmix_storage_persistence_t} type specifies different levels of persistence for a particular storage system.
+The \refstruct{pmix_storage_persistence_t} is a \code{uint64_t} type that defines a set of bit-mask flags for specifying different levels of persistence for a particular storage system.
 
```

Comment:
I think calling this out as a bitmask is fine, but if anyone is reviewing this this one is at least somewhat interesting compared to all the other constant declarations because we didn't specify this was a bitmask before and now we are.  I don't see a problem though.